ingestions/ingest_csvs.py

```diff
@@ -27,18 +30,79 @@
 PRIORITY = 1
 
 # initialize user credentials and the OOINet base URL
-BASE_URL = 'https://ooinet.oceanobservatories.org'
+# BASE_URL = 'https://ooinet.oceanobservatories.org'
+BASE_URL = 'https://ooinet-west.oceanobservatories.org'
+DEPLOY_URL = '12587/events/deployment/inv/'
 credentials = netrc.netrc()
-API_KEY, USERNAME, API_TOKEN = credentials.authenticators('ooinet.oceanobservatories.org')
```

Comment:
Do you want to keep this line commented out like you did above?

Reply:
That line is commented out so we can quickly switch between ooinet and ooinet-west when running ingestions. To switch from one to the other the appropriate line just has to get commented out